src/public/index.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <link rel="stylesheet" type="text/css" href="./styles/style.css">
     <meta charset='utf8'>
     <title>CodeWiki</title>
     <link rel="stylesheet" href="/lib/bootstrap.css">
     <link rel="stylesheet" href="/styles/font-awesome.css">
     <link rel="stylesheet" href="/styles/style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
What do we need jquery for? Is there a way to do whatever it is with a dependency we already have?